generate_parameter_library_py/generate_parameter_library_py/jinja_templates/parameter_library_header

```diff
@@ -113,6 +114,11 @@ struct StackParams {
       return params_;
     }
 
+    std::vector<rcl_interfaces::msg::ParameterDescriptor> get_descriptors(const std::vector<std::string> & names) const{
+      std::lock_guard<std::mutex> lock(mutex_);
+      return parameters_interface_->describe_parameters(names);
```

Comment:
I'm not sure we need this method. If it just calls `describe_parameters` on the node we can also just call that in the tests.